zenith_utils/src/pq_proto.rs

```diff
@@ -37,19 +38,13 @@ pub enum FeMessage {
 }
 
 #[derive(Debug)]
-pub struct FeStartupMessage {
-    pub version: u32,
-    pub kind: StartupRequestCode,
-    // optional params arriving in startup packet
-    pub params: HashMap<String, String>,
-}
-
-#[derive(Debug)]
-pub enum StartupRequestCode {
-    Cancel,
-    NegotiateSsl,
-    NegotiateGss,
-    Normal,
+pub enum FeInitialMessage {
```

Comment:
Consider `CancelRequest { backend_pid: u32, cancel_key: u32, }`
Same for `StartupMessage`